src/current/cockroachcloud/alerts-page.md

```diff
@@ -18,9 +18,9 @@ The **Alerts** page is applicable for CockroachDB {{ site.data.products.advanced
 
 [Org Administrators]({% link cockroachcloud/authorization.md %}#org-administrator) receive email alerts when:
 
-- A cluster is scheduled for an automatic [patch version upgrade]({% link cockroachcloud/upgrade-policy.md %}#patch-version-upgrades) and again after the upgrade is complete. 
-- When a cluster is scheduled for [maintenance]({% link cockroachcloud/advanced-cluster-management.md %}#set-a-maintenance-window) that could temporarily impact the cluster's performance. 
-- When a cluster's CockroachDB version is nearing [end of life](https://www.cockroachlabs.com/docs/releases/release-support-policy#support-cycle) and must be upgraded to maintain support.
+- A cluster is scheduled for an automatic [patch version upgrade]({% link cockroachcloud/upgrade-policy.md %}#patch-version-upgrades) and again after the upgrade is complete.
+- When a cluster is scheduled for [maintenance]({% link cockroachcloud/advanced-cluster-management.md %}#set-a-maintenance-window) that could temporarily impact the cluster's performance.
+- When a cluster's CockroachDB version is nearing [end of life]({% link releases/release-support-policy.md %}#support-cycle) and must be upgraded to maintain support.
```

Comment:
not sure where this link anchor is coming from, in https://github.com/cockroachdb/docs/pull/18875/files#diff-2e6f522d8fc3cc357a2551b2ad5112c5af53419fb52710f61e40c053bfabbab2R22 the anchors are updated to link to the `#support-phases` anchor which exists (this one does not)

It seems we should merge in the actual changes from #18875 which did not have this build error 

This broken link makes me think this is not a merge, but trying to manually add the edits?